collection-scripts/gather_main

```diff
@@ -21,8 +21,15 @@ procids+=($!)
 gather_dr_resources &
 procids+=($!)
 
-gather_odf_client &
-procids+=($!)
+storagecluster_remote=$(oc get storagecluster --all-namespaces  -o yaml | grep -i 'allowRemoteStorageConsumers')
```

Comment:
Storagecluster CRD won't be present in client cluster on a provider-client(provider mode) setup.

Reply:
@OdedViner please re-verify the check and paste the logs collected.
According to the check you mentioned, odf_client script will not run even if it is a client cluster.